sympy/algebras/quaternion.py

```diff
@@ -811,7 +812,8 @@ def axis(self):
         Explanation
         ===========
 
-        Given a quaternion $q = a + bi + cj + dk$ and returns $\mathbf{Ax}(q)$ ie normalized versor (vector part) of that quaternion.
+        Given a quaternion $q = a + bi + cj + dk$, returns $\mathbf{Ax}(q)$  i.e., the versor of the vector part of that quaternion
+        equal to $\mathbf{U}[\mathbf{V}(q)]$
```

Comment:
It's missing a dot at the end of this, to avoid mixing with "The axis is ..." that follows.